wiki/markdown/index.md

```diff
@@ -0,0 +1,18 @@
+## Introduction
+
+REMReM (**REST Mailbox for Registered Messages**) is a set of tools that can be used to generate validated Eiffel messages and publish them on a RabbitMQ message bus. They can be run as micro services or as stand-alone CLI versions. For more details on the micro services and the REMReM design, see [Eiffel REMReM](https://github.com/eiffel-community/eiffel-remrem).
+
+Eiffel REMReM Publish is a tool that can be used to publish Eiffel messages on a RabbitMQ message bus.
+
+## Compatibility
+
+*   JDK 8
+*   Tomcat 8
+*   RabbitMQ Server 3.6.x
+
+For supporting latest features, Eiffel REMReM Publish should use the latest version of [Eiffel REMReM Semantics](https://eiffel-community.github.io/eiffel-remrem-semantics/).
```

Comment:
The version of REMReM semantics will change so it's better to point to the GitHub repository instead, since it will have more up-to-date content than github pages.